src/data/ACTIONS/layers/patch6.1.ts

```diff
@@ -23,5 +23,17 @@ export const patch610: Layer<ActionRoot> = {
 				'FOURFOLD_FAN_DANCE',
 			],
 		},
+
+		// SMN 6.1 changes
+		SEARING_LIGHT: {
+			icon: 'https://xivapi.com/i/002000/002780.png',
+			statusesApplied: ['SEARING_LIGHT'],
+		},
+		RUBY_RITE: {
+			gcdRecast: 3000,
+		},
+		RUBY_CATASTROPHE: {
+			gcdRecast: 3000,
+		},
```

Comment:
Is this missing Ruby Disaster, Ruby Outburst, and Ruby Ruin III? I'm not familiar enough with lv90 SMN to know what's actually available vs replaced by traits, or just not used much since iirc Ruby/Emerald/Topaz Disaster are replaced with Tri-Disaster when you don't have a summon out or something?

Reply:
Those are all the lower level skills that are not currently in the xiva data.  I do plan to put those in at some point to make low level logs not so terrible.  Whenever I put the lower level skills in the base data, the ones you mentioned will get entries here at that point.

Reply:
That's fine, I'd not bother if they're lower level. Was hard to tell what's actually available at lv90 from the job guide page on Lodestone.